pkg/conn/conn.go

```diff
@@ -58,11 +64,15 @@ func pdGet(ctx context.Context, addr string, prefix string, cli *http.Client) ([
 		return nil, errors.Trace(err)
 	}
 	url := fmt.Sprintf("%s/%s", u, prefix)
-	req, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	var (
+		req  *http.Request
+		resp *http.Response
+	)
+	req, err = http.NewRequestWithContext(ctx, method, url, body)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	resp, err := cli.Do(req)
+	resp, err = cli.Do(req)
```

Comment:
Why removing the `:=`?

Reply:
okay, it's not necessary, change it back